### PR TITLE
Correct oauth2 and getting started typos

### PR DESCRIPTION
**Pull request by trodrigu**
I found a few typos while reviewing the docs.  I decided to submit a PR to fix!  Thanks for creating this framework.

### CODE REVIEW

File: documentation/tutorials/getting-started-with-authentication.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ used when converting tokens or session information into a resource record.
 This authentication strategy provides registration and sign-in for users using a local
 identifier (eg `username`, `email` or `phone_number`) and a password.  It will
 define register and sign-in actions on your "user" resource.  You are welcome to
-define either or both of these actions yourself if you wish to customise them -
```
Comment (jimsynz) [inline documentation/tutorials/getting-started-with-authentication.md:56]: This isn't a typo my guy 🤣

**Review by jimsynz — APPROVED** [review]
Thanks for your contribution!

thanks!

File: documentation/tutorials/getting-started-with-authentication.md (hunk 1 above)
Comment (trodrigu) [reply documentation/tutorials/getting-started-with-authentication.md]: Haha I just realized I've been using the British 🇬🇧 spelling.  Updated in 99bbcc4

Comment (jimsynz) [reply documentation/tutorials/getting-started-with-authentication.md]: 🙌

**Pull request merged**